src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2433,26 +2433,22 @@ Replication State Storage::
 `ReplicationPeerStorage` and `ReplicationQueueStorage`. The former one is for storing the
 replication peer related states, and the latter one is for storing the replication queue related
 states.
-  HBASE-15867 is only half done, as although we have abstract these two interfaces, we still only
-have zookeeper based implementations.
+  And in HBASE-27109, we have implemented the `ReplicationQueueStorage` interface to store the replication queue in the hbase:replication table.
 
 Replication State in ZooKeeper::
   By default, the state is contained in the base node _/hbase/replication_.
-  Usually this nodes contains two child nodes, the `peers` znode is for storing replication peer
-state, and the `rs` znodes is for storing replication queue state.
+  After 3.0.0, it only contains one child node, but before 3.0.0, we still use zk to store queue data.
```

Comment:
"Usually this nodes contains two child nodes, the `peers` znode is for storing replication peer state, and the `rs` znodes is for storing replication queue state. And if you choose the file system based replication peer storage, you will not see the `peers` znode. And starting from 3.0.0, we have moved the replication queue state to hbase:replication table, so you will not see the `rs` znode."